`Observations` property on series data gives static type checking error

Hi @pastorsj!

I've pulled series data using the `getObservationsForSeries` method and I'm trying to use the `filter` method on data accessed via the `observations` property. A static type checking error is blocking me from transpiling.

The code I'm using is:
```
const series = await fred.series.getObservationsForSeries('SP500');

let data = series.observations.filter((obj: any) => {
  // test implementation
});
```
And the error I'm getting is:
```
src/app.ts:31:36 - error TS2339: Property 'filter' does not exist on type 'ObservationSet'.

31     let data = series.observations.filter((obj: any) => {
                                      ~~~~~~
```
I double checked the wrapper documentation, and it does says that the returned JSON's `observations` property has a value of an array of objects, so it's odd that I'm not able to use an array method. 

Would appreciate it if you looked into this! Thanks in advance.

--- a/index.d.ts
+++ b/index.d.ts
@@ -40,7 +40,7 @@ declare namespace FredAPI {
     class Series {
         getSeries(seriesId: string, params?: object): Promise<SeriesSet>;
         getCategoriesForSeries(seriesId: string, params?: object): Promise<CategorySet>;
-        getObservationsForSeries(seriesId: string, params?: object): Promise<ObservationInfo>;
+        getObservationsForSeries(seriesId: string, params?: object): Promise<ObservationSet>;
         getReleaseForSeries(seriesId: string, params?: object): Promise<ReleaseSet>;
         getSeriesThatMatchesSearch(searchText: string, params?: object): Promise<SeriesInfo>;
         getTagsForSeriesSearch(seriesSearchText: string, params?: object): Promise<RelatedTagsInfo>;
@@ -199,16 +199,16 @@ declare namespace FredAPI {
         series: SeriesInfo[];
     }
 
-    interface ObservationInfo extends StandardInfo {
+    interface ObservationSet extends StandardInfo {
         observation_start: string;
         observation_end: string;
         units: string;
         output_type: number;
         file_type: string;
-        observations: ObservationSet;
+        observations: ObservationInfo[];
     }
 
-    interface ObservationSet extends TimeInfo {
+    interface ObservationInfo extends TimeInfo {
         date: string;
         value: string;
     }

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
     "name": "node-fred",
-    "version": "2.1.9",
+    "version": "2.1.10",
     "description": "Another nodejs wrapper for the st louis fed",
     "main": "lib/index.js",
     "types": "index.d.ts",

